### PR TITLE
Modified HoogleSearch in repository h.json to change ownership

### PR DESCRIPTION
**Pull request by pennychase**
- [ X] I'm the package's author and/or maintainer.
- [X ] I have have read [the docs][1].
- [X ] I have tagged a release with a [semver][2] version number.
- [X ] My package repo has a description and a README describing what it's for and how to use it.
- [X ] My package doesn't add context menu entries. *
- [ X] My package doesn't add key bindings. **
- [X ] Any commands are available via the command palette.
- [ ] Preferences and keybindings (if any) are listed in the menu and the command palette, and open in split view.
- [ ] If my package is a syntax it doesn't also add a color scheme. ***
- [ ] I use [.gitattributes][3] to exclude files from the package: images, test files, sublime-project/workspace.

My package is an updated version of Hoogle Search. The version on Package Control is broken. It was last updated 7 years ago and the Hoogle API has changed since then. I updated the package to work with the current Hoogle API and submitted a PR to the package author. The author hasn't responded and on Discord it was suggested that I submit on PR on Package Control to change the package to my fork.

[1]: https://packagecontrol.io/docs/submitting_a_package
[2]: https://semver.org
[3]: https://www.git-scm.com/docs/gitattributes#_export_ignore

### CODE REVIEW

**Comment by braver** [comment]
@s4wny just to be sure, are you ok with this?

**Comment by pennychase** [comment]
I had sent @S4wny an email, which he responded to today. He has made me a contributor to the repo, so I want to rescind the PR. Should I close the PR?

**Comment by braver** [comment]
Oh, that's perfect. If you can upgrade the existing package there is no need to continue this PR. However, you may want to edit (or remove) the author information here?

**Comment by pennychase** [comment]
I'm not sure what you mean by editing (or removing) the author information.

File: repository/h.json [hunk 1]
```diff
@@ -828,8 +828,8 @@
 		},
 		{
 			"name": "HoogleSearch",
-			"details": "https://github.com/s4wny/HoogleSearch",
-			"author": "Sawny",
+			"details": "https://github.com/pennychase/HoogleSearch",
+			"author": "Penny Chase",
```
Comment (braver) [inline repository/h.json:832]: If you want to keep the existing repo url in "details" you may still want to update "author" here.

**Pull request merged**